G/GoogleTest/build_tarballs.jl

```diff
@@ -3,7 +3,7 @@
 using BinaryBuilder, Pkg
 
 name = "GoogleTest"
-version = v"1.11.0"
+version = v"1.11.1"
```

Comment:
No need to change the version number for this change.  You could also update to a newer version, up to you